src/components/LeftPane/Drawers/AnalyticsDrawer/AnalyticsDrawer.tsx

```diff
@@ -1,6 +1,9 @@
 import { Drawer } from 'material-ui'
```

Comment:
What's the best practice for import ordering in js / react?

Reply:
We use https://palantir.github.io/tslint/rules/ordered-imports/ to do it for us :)